app/src/main/java/org/oppia/android/app/topic/TopicFragmentPresenter.kt

```diff
@@ -105,10 +106,12 @@ class TopicFragmentPresenter @Inject constructor(
         }
       }
     }.attach()
-    if (topicId.isNotEmpty() && storyId.isNotEmpty())
-      setCurrentTab(TopicTab.LESSONS)
-    else if (topicId.isNotEmpty() && storyId.isEmpty())
-      setCurrentTab(TopicTab.INFO)
+    if (isConfigChanged) {
```

Comment:
Negate this condition based on above suggestion:

Either: `if (!isConfigChanged) {` or `if (isConfigNotChanged) {`

Reply:
done